spec/services/proofing/resolution/plugins/aamva_plugin_spec.rb

```diff
@@ -92,6 +92,39 @@ def sp_cost_count_with_transaction_id
             expect { call }.to_not change { sp_cost_count_for_issuer }
           end
         end
+
```

Comment:
Played with this a little, and got it down to:
```suggestion
        context 'middle name is not included in first name' do
          let(:applicant_pii) do
            Idp::Constants::MOCK_IDV_APPLICANT_WITH_SSN.merge(first_name: 'FAKEY MIDDLE')
          end

          let(:failing_result) do
            Proofing::StateIdResult.new(
              success: false,
              vendor_name: 'state_id:aamva',
              transaction_id: proofer_transaction_id,
              errors: { first_name: 'UNVERIFIED' },
            )
          end

          it 're-calls the AAMVA proofer with the middle name removed from the first name field and tracks SP cost for both calls' do
            expect(plugin.proofer).to receive(:proof)
              .with(hash_including(first_name: 'FAKEY MIDDLE')).ordered.and_return(failing_result)
            expect(plugin.proofer).to receive(:proof)
              .with(hash_including(first_name: 'FAKEY')).ordered.and_return(proofer_result)

            expect { call }.to(change { sp_cost_count_with_transaction_id }.to(2))
          end
        end
```